node/src/components/event_stream_server/sse_server.rs

```diff
@@ -62,6 +62,11 @@ pub enum SseData {
         #[data_size(skip)]
         execution_result: Box<ExecutionResult>,
     },
+    Equivocation {
```

Comment:
Pretty sure clippy will tell you to put a comment on public API.

Reply:
Applied comment